app/lib/resources.server.ts

```diff
@@ -5,40 +5,32 @@ import { processMarkdown } from "./md.server";
 import resourcesYamlFileContents from "../../data/resources.yaml?raw";
 import { slugify } from "~/ui/primitives/utils";
 import type { Octokit } from "octokit";
+import type { ResourceYamlData } from "~/schemas/yaml-resource-schema";
+import { yamlResourceSchema } from "~/schemas/yaml-resource-schema";
 
 export type CacheContext = { octokit: Octokit };
 
 const GITHUB_URL = "https://github.com";
 
-// TODO: parse this with zod
-let _resources: ResourceYamlData[] = yaml.parse(resourcesYamlFileContents);
+let _resources = yamlResourceSchema.parse(
+  yaml.parse(resourcesYamlFileContents),
+);
```

Comment:
Mind explicitly running this in a test? Probably in `resources.server.test.ts`

Right now we're inadvertently testing it, because in `resources.server.test.ts` we import `replaceRelativeLinks`, which ends up running this. However, if it fails it's going to be a bit confusing. Better to make it explicit with a test

And while we're here 🙈 let's put `resources.server.ts` and `resources.server.test.ts` in a directory together, just so they're a little more easily colocated

Something like `resources.server/index.ts` and `resources.server/resources.test.ts` should probably be good 🙏

Reply:
done